src/qtiCommonRenderer/tpl/interactions/associateInteraction.pair.tpl

```diff
@@ -1,4 +1,4 @@
-<li class="{{#if empty}}incomplete-pair{{/if}}"{{#if attributes.xml:lang}} lang="{{attributes.xml:lang}}"{{/if}}>
+<li class="hidden{{#if empty}} incomplete-pair{{/if}}"{{#if attributes.xml:lang}} lang="{{attributes.xml:lang}}"{{/if}}>
```

Comment:
**nitpick:** the `display:none` was applied to the `incomplete-pair` class, which is not always added. I'd rather suggest adding the `hidden` class following the same rules. Unless the pair needs to be always hidden at the beginning.

Reply:
reasoned mention, fixed